clients/web/src/templates/widgets/searchModeSelect.pug

```diff
@@ -6,7 +6,4 @@ for mode in modes
       input.g-search-mode-radio(
           type="radio", name="search-mode", value=mode,
           checked=(mode === currentMode ? 'checked' : undefined))
-      if mode === 'prefix'
-        |  Search by prefix
-      else if mode === 'text'
-        |  Full text search
+      = descriptionModes(mode)
```

Comment:
I'm not sure it's possible, take a look at [this issue](https://stackoverflow.com/questions/38295332/pug-jade-input-is-a-self-closing-element-input-but-contains-nested-conten)

Reply:
Ah, I see, I thought that the content was within the `input` as text, and the style was misleading, but I forgot `input` was self-closing. So, this is just plain text that's being injected.

Can you refactor this code so that the text is directly inside a normal `label`, and the parent element is a `div`? It doesn't have to be exactly this, but the current hierarchy doesn't make much sense when compiled.